Formula/podman.rb

```diff
@@ -4,36 +4,69 @@ class Podman < Formula
   license "Apache-2.0"
 
   stable do
-    url "https://github.com/containers/podman/archive/v3.4.4.tar.gz"
-    sha256 "718c9e1e734c2d374fcf3c59e4cc7c1755acb954fc7565093e1d636c04b72e3a"
+    url "https://github.com/containers/podman/archive/v4.0.2.tar.gz"
+    sha256 "cac4328b0a5e618f4f6567944e255d15fad3e1f7901df55603f1efdd7aaeda95"
+
+    patch do
+      url "https://github.com/containers/podman/commit/cdb6deb148f72cad9794dec176e4df1b81d31d08.patch?full_index=1"
+      sha256 "10d1383f4179fd4af947f554677c301dc64c53c13d2f0f59aa7b4d370de49fcf"
+    end
+    patch do
+      url "https://fedorapeople.org/groups/podman/testing/darwin_qemu_search_paths.patch"
+      sha256 "6315f2c8071b0bdba5c3346c4581aaed70696a7e63c3a361a1e1bd78eb5a3f51"
+    end
```

Comment:
Has this been send upstream too?

Reply:
yes. https://github.com/containers/common/pull/950

Reply:
Yes, already merged upstream!